pkg/kubelet/config/file_unsupported.go

```diff
@@ -19,8 +19,11 @@ limitations under the License.
 // Reads the pod configuration from file or a directory of files.
 package config
 
-import "errors"
+import (
+	"errors"
+	"time"
+)
 
-func (s *sourceFile) watch() error {
-	return errors.New("source file is unsupported in this build")
+func (s *sourceFile) watch() {
+	// watching source file is unsupported in this build
```

Comment:
I still don't see why you wouldn't want to log an error, so anyone using watch on non-linux platform will be alerted that the watch they are using is not implemented.